src/applications/facility-locator/actions/index.js

```diff
@@ -501,6 +501,7 @@ export const getProviderSpecialties = () => async dispatch => {
 };
 
 export const geolocateUser = () => async dispatch => {
+  const GEOLOCATION_TIMEOUT = 10000;
```

Comment:
can we add this in our constants file ?

Reply:
I think it's OK to leave it here since it's only used here.